guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -166,21 +109,68 @@ Finally, you can calculate your estimated generated revenue share by multiplying
 
-- Virtual Price = Sum of all JLP Assets (in USD) / Total Quantity of JLP in circulation
-- Market Price = Virtual Price + Market-assigned Premium (when AUM Limit is hit)
+2. **Profit and Loss (PnL) Dynamics**
 
-Usually, users can mint new JLP or redeem (burn) them at the Virtual Price. However, when AUM Limits are hit, new minting of JLP is disabled to cap the amount of TVL in the pool.
+Traders' PnL from perpetual trading impacts the JLP pool. If a trader incurs a net positive PnL, the losses are sourced from the JLP pool to compensate the trader. Conversely, if a trader's PnL is a net negative, the gains are deposited into the JLP pool for LP holders.
 
-When this happens, the demand for JLP on the market usually leads to a premium for JLP compared to the virtual price.
+**Long Trade Scenario:** If the trader profits on the long, the JLP pool will lose in token quantity but not in USD value because the underlying token value in the pool appreciates in value as well.
 
-You may sell your JLP for the Market Price at any time. If the Market Price is below the Virtual Price, your JLP tokens are redeemed (burned) at the virtual price instead of sold at the market price.
+![image](jlp-long-scenarios.png)
 
-![image](../img/jlp/jlp-TVL.png)
+**Short Trade Scenario:** If the trader profits on the short, the JLP pool will lose some of the stablecoins but the shorted token will remain the same. This causes a net USD loss on the Pool.
+
+![image](jlp-short-scenarios.png)
+
+*This [research article](https://skribr.io/app/article/exploring-jupiters-perpetual-futures-a-comprehensive-research-analysis/) from LeeWay provides more detail on how this works.*
+
+3. **Token Price Fluctuations:** The JLP pool consists of both stable and non-stable tokens. Fluctuations in token prices can affect the value of JLP. As a result, users may find that their withdrawn tokens can be worth less compared to their initial deposit. Additionally, deposit and withdrawal fees for the JLP Pool may further reduce the number of tokens withdrawn, particularly for shorter holding periods.
+
+**Can JLP go down?** Yes. As shown in the trader profiting on a short trade, JLP in USD value will go down when the fees generated are lower than depreciation of assets and payout from traders' profit.
+
+### Pool Weightage
+
+In the JLP pool, each asset is assigned a **target weight**, which is set by the team. 
+
+![jlp-weightage](jlp-weightage.png)
+
+Experts like [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) conduct regular risk assessments and market analyses in response to the evolving market conditions, and discuss with the community before presenting to the team for rollout. 
```

Comment:
```suggestion
Jupiter works with [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) to conduct regular risk assessments and market analyses in response to evolving market conditions, and discuss with the community before presenting to the team for rollout. 
```

I think we can change the first part to say `Jupiter works with...`; do you mind helping me change the last part of the paragraph as well? Since it reads kinda weird now that I've changed the first line, and IDK how to make it sound better 😅